aspnetcore/host-and-deploy/docker/visual-studio-tools-for-docker.md

```diff
@@ -50,7 +50,7 @@ The Visual Studio Tools for Docker don't support adding Docker to an existing AS
 
 ## Docker assets overview
 
-The Visual Studio Tools for Docker add a *docker-compose* project to the solution, containing the following:
+The Visual Studio Tools for Docker add a *docker-compose* project to the solution, containing the following files:
```

Comment:
I think that participle phrase at the end is restrictive because it tells the reader what the items are (esp. now that you're clarifying it), so I recommend dropping the comma.